src/tests/ascent/t_ascent_rover.cpp

```diff
@@ -211,8 +213,8 @@ TEST(ascent_rover, test_xray_blueprint_braid_lowres)
     extracts["e1/params/blueprint"] = "json";
 
     // Output resolution
-    extracts["e1/params/width"] = 11;
-    extracts["e1/params/height"] = 11;
+    extracts["e1/params/camera/image_width"] = 11;
+    extracts["e1/params/camera/image_height"] = 11;
```

Comment:
We might have to change these names. Let's discuss another time. Not a problem for this PR.